config/locales/cy.yml

```
@@ -496,7 +496,7 @@ cy:
         about: Amdanom ni
         about_our_services: Gwybodaeth am ein gwasanaethau
         access_and_opening: Mynediad i'n swyddfeydd ac amseroedd agor
-        accessible_documents_policy:
+        accessible_documents_policy: olisi Dogfennau Hygyrch
```

I think you missed a letter! :smile:

```suggestion
        accessible_documents_policy: Polisi Dogfennau Hygyrch
```

---

No I didn't 👀